rai_test_utils/requirements.txt

```diff
@@ -4,4 +4,4 @@ scikit-learn
 scipy
 shap
 lightgbm
-xgboost
+xgboost
```

Comment:
What is different in this line?

Reply:
Honestly I had the same question lol, looks like a newline tweak, so restored it from main.